src/addon/info.cpp

```diff
@@ -233,13 +233,13 @@ std::string addon_info::display_icon() const
 {
 	std::string ret = icon;
 
-	// make sure it's set to something when there are issues
-	// otherwise display errors will spam the log while the add-ons manager is open
 	if(ret.empty()) {
-		ret = "misc/blank-hex.png";
-	} if(!image::exists(image::locator{ret}) && !ret.empty()) {
+		// The canvas class will log (at debug level) that this is an empty string,
+		// but it will handle the empty string without problems.
```

Comment:
If this is adding more comments for that one presumably useless debug log message... 
Let's just remove all the nonsense. Having comments depend on what some debug level message of some other code does is just asking to get outdated in no time.

Reply:
Agreed. Just revert the change to this function from d9ef464b1e9446b3a53e2d3bd86635529c897df4.